Readability.js

```diff
@@ -1141,7 +1141,7 @@ Readability.prototype = {
           this._attempts.push({articleContent: articleContent, textLength: textLength});
           // No luck after removing flags, just return the longest text we found during the different loops
           this._attempts.sort(function (a, b) {
-            return a.textLength < b.textLength;
+            return (a.textLength < b.textLength) ? 1 : -1;
```

Comment:
Thanks for the patch! How about:

```suggestion
            return b.textLength - a.textLength;
```

Does that also work?

Reply:
Yes, that also works. Please feel free to go with whichever you think is clearer.

Reply:
Hm, @jemrobinson seems that I can't accept my own suggestion because I (of course!) don't have permission to push to your clone of the repo - would you mind accepting it so I can just merge this from the web UI? Sorry for the confusion, I'll give github some feedback on this...

Reply:
OK, I *think* that should be done now. I also realised that your solution deals with the case `a.textLength == b.textLength` which the original code doesn't, so it's definitely better. Thanks for reviewing this!